src/components/modal/modal.tsx

```diff
@@ -394,9 +400,14 @@ export class Modal implements ConditionalSlotComponent, OpenCloseComponent {
   };
 
   @Watch("active")
+  activeHandler(value: boolean): void {
+    this.open = value;
+  }
+
   @Watch("open")
   async toggleModal(value: boolean, oldValue: boolean): Promise<void> {
     if (value !== oldValue) {
```

Comment:
Yeah i'm not sure if this check is needed. It shouldn't be if the watcher is the only thing calling it.